sources/packages/backend/apps/api/src/app.institutions.module.ts

```diff
@@ -45,7 +45,9 @@ import {
   StudentInstitutionsController,
   OverawardInstitutionsController,
   StudentControllerService,
+  RestrictionInstitutionController,
```

Comment:
client type should be plural for both file and controller name `RestrictionInstitutionsController` and the controller file should be `/restriction.institutions.controller.ts`